scripts/deploy.sh

```
@@ -0,0 +1,159 @@
+sync_env() {
+    local file=$1
+    if [ ! -f "$file" ]; then
+        if [ -f ".env.example" ]; then
+            cp .env.example "$file"
+        else
+            touch "$file"
+        fi
+    fi
+
+    # Update values (using a simple temp file to be portable)
+    sed -i.bak "s|^PUBLIC_STELLAR_NETWORK=.*|PUBLIC_STELLAR_NETWORK=$(echo $STELLAR_NETWORK | tr '[:lower:]' '[:upper:]')|" "$file"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)